lib/requester/core.js

```diff
@@ -28,7 +28,21 @@ var _ = require('lodash'),
 
     GET = 'get',
 
-    METHODS_WITHOUT_BODY = ['get', 'copy', 'head', 'purge', 'unlock'],
+    /**
+     * List of request methods without body.
+     *
+     * @private
+     * @type {Object}
+     *
+     * @note hash is used to reduce the lookup cost
+     */
```

Comment:
Add a comment explaining the source of this hash.

Reply:
Also, this could get conflated with custom request methods that 'should not' have a body.